test/unit/feature_test.rb

```diff
@@ -62,6 +79,17 @@
       _(logger_io.string).must_match(/WARN/)
       _(logger_io.string).must_match(/test-feature-nonesuch/)
     end
+
+    it "should not give warnings for inspec custom plugins" do
```

Comment:
Could you add another test where it gives warning, executing line 17 of runner.rb?

Reply:
Ignore, if we already have such a test already.

Reply:
Yes it's already there.

Reply:
Ref: https://github.com/inspec/inspec/pull/6988/files#diff-2c333ccd448d35de740f548d16f093e60c52c9036914b72ce0ecbb654b934f2bR79